### PR TITLE
Add Teros forcing data for 2022

### PR DESCRIPTION
**Pull request by stephpenn1**
image

### CODE REVIEW

File: tempest_teros.R [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+Sys.setenv(TZ='EST')
+
+# List files
+files <- list.files(path = "tempest_data", pattern = "PNNL_1", full.names = T) #filter for only Control plot
+
+# Define function to read in teros data and format
+read_teros <- function(filename){
+
+  message(paste("processing", filename))
```
Comment (bpbond) [inline tempest_teros.R]: FYI you don't need the paste; `message` can take multiple arguments.

**Review by bpbond — APPROVED** [review]
Nice!

**Comment by stephpenn1** [comment]
Would like to plot control plot by depth to see where this variation is coming from before merging

**Comment by stephpenn1** [comment]
The variation is pretty similar in the whole 30cm profile, so I feel confident keeping as is and averaging across the 5-30cm profile

image

**Pull request merged**